### PR TITLE
Added Query Implementation

### PR DESCRIPTION
**Pull request by enhanse**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

File: packages/cubejs-testing-shared/src/db/vertica.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export class VerticaDBRunner extends DbRunnerAbstract {
 
     const container = new GenericContainer(`vertica/vertica-ce:${version}`)
       .withEnv('VERTICA_DB_NAME', 'test')
+      // .withEnv('VERTICA_MEMDEBUG', '2') // Required for ARM architecture
```
Comment (timbrownls20) [inline packages/cubejs-testing-shared/src/db/vertica.ts:16]: best to remove - or control with const or something. Legit probably for mac testing

File: packages/cubejs-vertica-driver/src/VerticaQuery.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    const p = (!type || type === 'contains' || type === 'ends') ? '%' : '';
+    const s = (!type || type === 'contains' || type === 'starts') ? '%' : '';
+    return ` ILIKE (${column}${not ? ' NOT' : ''}, CONCAT('${p}', ${this.allocateParam(param)}, '${s}'))`;
+  }
+
+  castParameter() {
+    if (this.definition().type === 'boolean') {
+      return 'CAST(? AS BOOLEAN)';
+    } else if (this.measure || this.definition().type === 'number') {
+      // TODO here can be measure type of string actually
```
Comment (timbrownls20) [inline packages/cubejs-vertica-driver/src/VerticaQuery.js:26]: remove TODO

**Review by timbrownls20 — CHANGES_REQUESTED** [review]

**Review by timbrownls20 — COMMENTED** [review]
i think it's best to update the vertica driver and bring in the latest version of cubejs into this PR

File: packages/cubejs-testing-shared/src/db/vertica.ts (hunk 1 above)
Comment (enhanse) [reply packages/cubejs-testing-shared/src/db/vertica.ts]: Removed

**Review by timbrownls20 — APPROVED** [review]

**Review by timbrownls20 — APPROVED** [review]

**Pull request merged**